diff -urN kmediafactory-0.4.1.orig/player/kmfplayerlayout.ui kmediafactory-0.4.1/player/kmfplayerlayout.ui
--- kmediafactory-0.4.1.orig/player/kmfplayerlayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/player/kmfplayerlayout.ui	2005-10-29 11:41:22.000000000 +0000
@@ -624,24 +624,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kxinewidget.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>positionslider.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/dvimport/progresslayout.ui kmediafactory-0.4.1/plugins/dvimport/progresslayout.ui
--- kmediafactory-0.4.1.orig/plugins/dvimport/progresslayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/dvimport/progresslayout.ui	2005-10-29 11:41:34.000000000 +0000
@@ -118,7 +118,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/output/dvdinfolayout.ui kmediafactory-0.4.1/plugins/output/dvdinfolayout.ui
--- kmediafactory-0.4.1.orig/plugins/output/dvdinfolayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/output/dvdinfolayout.ui	2005-10-29 11:44:40.000000000 +0000
@@ -168,11 +168,4 @@
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klistview.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/template/preview_1/settings.ui kmediafactory-0.4.1/plugins/template/preview_1/settings.ui
--- kmediafactory-0.4.1.orig/plugins/template/preview_1/settings.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/template/preview_1/settings.ui	2005-10-29 11:43:20.000000000 +0000
@@ -165,16 +165,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kcolorcombo.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/template/preview_3/settings.ui kmediafactory-0.4.1/plugins/template/preview_3/settings.ui
--- kmediafactory-0.4.1.orig/plugins/template/preview_3/settings.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/template/preview_3/settings.ui	2005-10-29 11:43:32.000000000 +0000
@@ -165,16 +165,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kcolorcombo.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/template/preview_6/settings.ui kmediafactory-0.4.1/plugins/template/preview_6/settings.ui
--- kmediafactory-0.4.1.orig/plugins/template/preview_6/settings.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/template/preview_6/settings.ui	2005-10-29 11:43:43.000000000 +0000
@@ -176,16 +176,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kcolorcombo.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/template/simple/settings.ui kmediafactory-0.4.1/plugins/template/simple/settings.ui
--- kmediafactory-0.4.1.orig/plugins/template/simple/settings.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/template/simple/settings.ui	2005-10-29 11:43:55.000000000 +0000
@@ -184,16 +184,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kcolorcombo.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/template/templateconfiglayout.ui kmediafactory-0.4.1/plugins/template/templateconfiglayout.ui
--- kmediafactory-0.4.1.orig/plugins/template/templateconfiglayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/template/templateconfiglayout.ui	2005-10-29 11:44:27.000000000 +0000
@@ -60,7 +60,4 @@
     <include location="local" impldecl="in implementation">kmflanguagewidgets.h</include>
 </includes>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>kmflanguagewidgets.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/template/travel/settings.ui kmediafactory-0.4.1/plugins/template/travel/settings.ui
--- kmediafactory-0.4.1.orig/plugins/template/travel/settings.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/template/travel/settings.ui	2005-10-29 11:44:06.000000000 +0000
@@ -146,13 +146,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kcolorcombo.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/video/autochapterslayout.ui kmediafactory-0.4.1/plugins/video/autochapterslayout.ui
--- kmediafactory-0.4.1.orig/plugins/video/autochapterslayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/video/autochapterslayout.ui	2005-10-29 11:43:06.000000000 +0000
@@ -176,9 +176,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>ktimewidget.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/video/chapterslayout.ui kmediafactory-0.4.1/plugins/video/chapterslayout.ui
--- kmediafactory-0.4.1.orig/plugins/video/chapterslayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/video/chapterslayout.ui	2005-10-29 11:41:52.000000000 +0000
@@ -557,18 +557,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kmfchapterlistview.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmfimageview.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/video/languageselectionlayout.ui kmediafactory-0.4.1/plugins/video/languageselectionlayout.ui
--- kmediafactory-0.4.1.orig/plugins/video/languageselectionlayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/video/languageselectionlayout.ui	2005-10-29 11:42:06.000000000 +0000
@@ -138,9 +138,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kmflanguagewidgets.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/video/subtitleoptionslayout.ui kmediafactory-0.4.1/plugins/video/subtitleoptionslayout.ui
--- kmediafactory-0.4.1.orig/plugins/video/subtitleoptionslayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/video/subtitleoptionslayout.ui	2005-10-29 11:42:33.000000000 +0000
@@ -267,13 +267,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kmflanguagewidgets.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmffontchooser.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/video/videoconfiglayout.ui kmediafactory-0.4.1/plugins/video/videoconfiglayout.ui
--- kmediafactory-0.4.1.orig/plugins/video/videoconfiglayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/video/videoconfiglayout.ui	2005-10-29 11:42:19.000000000 +0000
@@ -88,8 +88,4 @@
 </includes>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kmflanguagewidgets.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/plugins/video/videooptionslayout.ui kmediafactory-0.4.1/plugins/video/videooptionslayout.ui
--- kmediafactory-0.4.1.orig/plugins/video/videooptionslayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/plugins/video/videooptionslayout.ui	2005-10-29 11:42:47.000000000 +0000
@@ -388,18 +388,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmflanguagewidgets.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/src/kmfoptionslayout.ui kmediafactory-0.4.1/src/kmfoptionslayout.ui
--- kmediafactory-0.4.1.orig/src/kmfoptionslayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/src/kmfoptionslayout.ui	2005-10-29 11:39:13.000000000 +0000
@@ -109,9 +109,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/src/logviewlayout.ui kmediafactory-0.4.1/src/logviewlayout.ui
--- kmediafactory-0.4.1.orig/src/logviewlayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/src/logviewlayout.ui	2005-10-29 11:40:42.000000000 +0000
@@ -113,8 +113,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>ktextbrowser.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/src/mediapagelayout.ui kmediafactory-0.4.1/src/mediapagelayout.ui
--- kmediafactory-0.4.1.orig/src/mediapagelayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/src/mediapagelayout.ui	2005-10-29 11:40:02.000000000 +0000
@@ -110,8 +110,4 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>ktoolbar.h</includehint>
-    <includehint>kmficonview.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/src/outputpagelayout.ui kmediafactory-0.4.1/src/outputpagelayout.ui
--- kmediafactory-0.4.1.orig/src/outputpagelayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/src/outputpagelayout.ui	2005-10-29 11:39:43.000000000 +0000
@@ -269,12 +269,4 @@
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kmficonview.h</includehint>
-    <includehint>kmfprogresslistview.h</includehint>
-    <includehint>kprogress.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/src/projectoptionslayout.ui kmediafactory-0.4.1/src/projectoptionslayout.ui
--- kmediafactory-0.4.1.orig/src/projectoptionslayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/src/projectoptionslayout.ui	2005-10-29 11:38:44.000000000 +0000
@@ -188,11 +188,4 @@
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-</includehints>
 </UI>
diff -urN kmediafactory-0.4.1.orig/src/templatepagelayout.ui kmediafactory-0.4.1/src/templatepagelayout.ui
--- kmediafactory-0.4.1.orig/src/templatepagelayout.ui	2005-10-29 11:38:23.000000000 +0000
+++ kmediafactory-0.4.1/src/templatepagelayout.ui	2005-10-29 11:40:22.000000000 +0000
@@ -146,8 +146,4 @@
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
 <layoutfunctions spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
-<includehints>
-    <includehint>kmficonview.h</includehint>
-    <includehint>kmfimageview.h</includehint>
-</includehints>
 </UI>
